fcrepo-connector-file/pom.xml

```diff
@@ -21,6 +21,12 @@
       <groupId>javax.jcr</groupId>
       <artifactId>jcr</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>org.modeshape</groupId>
+      <artifactId>modeshape-jcr</artifactId>
+      <scope>provided</scope>
```

Comment:
Right, right. In theory, we want the only transitive dependency to be "pure" JCR API, not impl APIs, right?